pom.xml

```diff
@@ -136,6 +136,11 @@
             <optional>true</optional>
         </dependency>
         <dependency>
+		    <groupId>commons-codec</groupId>
```

Comment:
@popprem we are not allowed to use external dependency with compile scope. We promised that to our clients.